### ISSUE TITLE
SVG icons id should be prefixed

### ISSUE THREAD

**Issue by tonifisler** [frontend/toolbox-reader#36]
We should prefix Toolbox icons to avoid duplicated ids with the custom icons from the project.

**Not like this:**
```html
<symbol id="icon-code" viewbox="0 0 114 102">
```

**Like this:**
```html
<symbol id="tlbx-icon-code" viewbox="0 0 114 102">
```

https://github.com/frontend/toolbox-reader/blob/ed1eabf9f696b2ea43fb778abff1a99d6d185fbf/src/components/Icon/Icons.js#L5-L39
